src/content/en/2020/resource-hints.md

```diff
@@ -5,19 +5,367 @@ chapter_number: 21
+  content="33%",
+  classes="big-number",
+  sheets_gid="1805612941",
+  sql_file="adoption.sql"
+) }}
+
+The relative popularity of `dns-prefetch` is unsurprising as it first appeared in 2009, and has the widest support out of all major Resource Hints. It had a 4% increase in Desktop adoption, compared to [2019](https://almanac.httparchive.org/en/2019/resource-hints#resource-hints). We saw a similar increase for `preconnect` as well. One key reason this was the largest growth between all hints, is the clear and useful advice the Lighthouse audit is giving on this matter. Starting from this year's report we also introduce how the latest dataset performs against Lighthouse recommendations.
+
+`preload` usage has had a slower growth with only a 2% increase from 2019. This could be in part because it requires a bit more attention. While you only need the domain to use `dns-prefetch` and `preconnect`, you must specify the resource to use `preload`. While you can safely assume `dns-prefetch` and `preconnect` will improve performance, if `preload` is used on the wrong resource, it can actually damage performance.
+
+`prefetch` is used by 3% of sites on Desktop, making it the least widely used resource hint. This low usage may be explained by the fact that `prefetch` is useful for improving subsequent—rather than current—page loads. Thus, it will be overlooked if a site is only focused on improving its landing page, or the performance of the first page viewed. In the coming years with a more clear definition on what to measure for improving subsequent page experience, it could help teams prioritize `prefetch` adoption with clear performance quality goals to reach.
+
+### Hints per page
+
+Across the board developers are learning how to better use Resource Hints, and compared to [2019](https://almanac.httparchive.org/en/2019/resource-hints#resource-hints) we've seen an improved use of `preload`, `prefetch` and `preconnect`. For expensive operations like preload and preconnect the median usage on desktop decreased from 2 to 1. We have seen the opposite for loading future resources with a lower priority with `prefetch`, with an increase from 1 to 2 in median per page.
+
+{{ figure_markup(
+  image="TODO",
+  caption="TODO",
+  description="TODO",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vTYAbLxN40s6mNR1jo0XDe_V4siN8TAsx2mryMp5IQmlJ-9O9eJxYROz7Rw6ozyFP6hlIZHxxh95GqX/pubchart?oid=320451644&format=interactive",
+  sheets_gid="175042082",
+  sql_file="hints_per_page.sql"
+) }}
+
+Resource hints are most effective when they're used selectively ("when everything is important, nothing is"). Having a more clear definition of what resources help improve Largest Contentful Paint moved the focus away from using `preconnect` and more towards `prefetch` by shifting some of the resource prioritization and freeing up bandwidth for what most helps the user at first.
```

Comment:
Yeah I can tell my point is not clear at all here. What I am trying to say it's less about the kind of resource but more about the timing when we make the prioritization. A developer that use `preload` for everything it could end up to damage LCP because we are over-prioritizing resources that are not important for LCP where instead we should move some of those late resources to a later stage by using `prefetch`.

I think mentioning `prefetch` here creates confusion, we can def drop it. But I still believe we should teach developers to be aware of the network timeline on when those resources are need it and how they impact LCP.

Reply:
Oh OK. The specific use of LCP confused me. But you're using that to mean "initial rendering" I think?

What about something like this then?

```suggestion
Resource hints are most effective when they're used selectively ("when everything is important, nothing is"). Having a more clear definition of what resources help improve critical rendering, versus future navigation optimizations, can move the focus away from using `preconnect` and more towards `prefetch` by shifting some of the resource prioritization and freeing up bandwidth for what most helps the user at first.
```